app/models/runtime/droplet_model.rb

```diff
@@ -138,7 +138,7 @@ def docker_user
         end
       end
 
-      container_user.presence || AppModel::DEFAULT_DOCKER_CONTAINER_USER
+      container_user.presence || (Config.config.get(:allow_process_root_user) ? AppModel::DEFAULT_DOCKER_CONTAINER_USER : AppModel::DEFAULT_CONTAINER_USER)
```

Comment:
This is semantically a bit confusing, since it means in some cases the `docker_user` default is not the `DEFAULT_DOCKER_USER`, but is instead the `DEFAULT_CONTAINER_USER`. Also we have to duplicate the same ternary in both droplet and process model.

Maybe instead (roughly):
1. Encapsulate the ternary in a method (on AppModel?) `default_docker_user`
1. `DEFAULT_DOCKER_CONTAINER_USER` goes away (since it's only used in one place)

Reply:
Indeed, I could see trying to move this up higher in the hierarchy in general to avoid duplication. I'll give it a try and see how it looks.